stock_average_daily_sale/models/stock_average_daily_sale.py

```diff
@@ -114,7 +127,7 @@ def _check_view(self):
         except Exception as e:
             raise e
         finally:
-            new_self.env.cr.close()
+            cr.close()
```

Comment:
@lmignon Is this really wanted (the code before was closing the new created cursor, not the current one)?

Reply:
The code close the new cursor...

```
cr = registry(self._cr.dbname).cursor()
....

cr.close()
```

I'll clean the code to improve readability

```python
    @api.model
    def _check_view(self):
        cr = registry(self._cr.dbname).cursor()
        with closing(cr):
            try:
                return self._check_materialize_view_populated(cr)
            except ObjectNotInPrerequisiteState:
                _logger.warning(
                    _("The materialized view has not been populated. Launch the cron.")
                )
                return False
            except Exception as e:
                raise e

```